ui/app/mainui/activitycenter/views/ActivityNotificationMessage.qml

```diff
@@ -2,57 +2,111 @@ import QtQuick 2.14
 import QtQuick.Layouts 1.14
 
 import StatusQ.Core 0.1
+import StatusQ.Core.Utils 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Components 0.1
 
 import shared 1.0
-import utils 1.0
-import shared.views.chat 1.0
 
 ActivityNotificationBase {
     id: root
 
-    property var messageContextMenu
+    readonly property string timestampString: Qt.formatTime(new Date(notification.timestamp), "hh:mm")
```

Comment:
```suggestion
    readonly property string timestampString: new Date(notification.timestamp).toLocaleTimeString()
```

Reply:
Same here pls :) https://doc.qt.io/qt-5/qml-qtqml-date.html#toLocaleTimeString-method

(you can use the `Locale.ShortFormat` here)

Reply:
fixed